.github/workflows/remoteconfig.yml

```diff
@@ -59,19 +59,15 @@ jobs:
 
     strategy:
       matrix:
-        # TODO: macos tests are blocked by https://github.com/erikdoe/ocmock/pull/532
-        target: [ios, tvos, macos --skip-tests, watchos]
+        # TODO: add watchos back
+        target: [ios, tvos, macos --skip-tests]
         podspec: [FirebaseRemoteConfig.podspec]
         os: [macos-14]
         include:
-          - os: macos-14
-            xcode: Xcode_15.3
-            # TODO(#13078): Fix testing infra to enforce warnings again.
-            tests: --allow-warnings
           # Flaky tests on CI
           - os: macos-14
             xcode: Xcode_16
```

Comment:
We might want to get ahead of https://github.com/actions/runner-images/issues/10703 and switch this to `macos-15` here and in `main`.

Reply:
Agreed, but punting for now.